sunpy/instr/aia/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+__author__ = 'tdwilkinson'
+
+"""
+SunPy
+=====
+
+An open-source Python library for Solar Physics data analysis.
+
+AIA response functions are ...
+
+Web Links
+---------
+Homepage: http://www.sunpy.org
+Documentation: http://docs.sunpy.org
+"""
+
+from .aiaprep import aiaprep
+
```

Comment:
Add `from .response import Response` here so that the class is accessible via 
```Python
import sunpy.instr.aia
response = sunpy.instr.aia.Response()
```